### PR TITLE
bump to v0.3.14

### PR DESCRIPTION
**Pull request by randy3k**
Will merge after CRAN accepts the submission.

### CODE REVIEW

**Comment by randy3k** [comment]
@renkun-ken CRAN check doesn't like the use of attach, PTAL.

File: R/diagnostics.R [hunk 1]
```diff
@@ -85,12 +85,12 @@ diagnose_file <- function(uri, content, is_rmarkdown = FALSE, globals = NULL, ca
     }
 
     if (length(globals)) {
-        env_name <- "languageserver:globals"
-        attach(globals, name = env_name, warn.conflicts = FALSE)
-        on.exit(detach(env_name, character.only = TRUE))
+        parent.env(globals) <- environment()
+        lints <- with(globals, lintr::lint(path, cache = cache, text = content))
```
Comment (renkun-ken) [inline R/diagnostics.R:89]: What if `globals` contains symbols such as `path`, and cache, and `content`?

Comment (renkun-ken) [reply R/diagnostics.R:89]: I tried putting `path <- NULL` in a package R file, and the following occurs:

```
Failed to run diagnostics: ! error in callr subprocess
Caused by error in `if (needs_tempfile) { ...`:
! missing value where TRUE/FALSE needed
```

Comment (randy3k) [reply R/diagnostics.R:89]: Good catch. Let me fix it.

Comment (renkun-ken) [reply R/diagnostics.R:89]: Looks like the variables must be accessible from the global environment. Looks like `globals()` must be attached and we'll have to walk-around the CRAN policy?

Comment (renkun-ken) [reply R/diagnostics.R:89]: Just like what callr has done at 

https://github.com/r-lib/callr/blob/5adbb27027c377b3ad0d4b3b471f81b36b02da67/R/hook.R#L7

Comment (randy3k) [reply R/diagnostics.R]: +1

Comment (renkun-ken) [reply R/diagnostics.R]: Already fixed

**Review by renkun-ken — APPROVED** [review]
LGTM

File: R/diagnostics.R (hunk 1 above)
Comment (randy3k) [reply R/diagnostics.R]: Thanks, submitting to CRAN.

**Comment by randy3k** [comment]
It's on the way to CRAN.

**Pull request merged**